### ISSUE TITLE
Support escapes in tag parsing

### ISSUE THREAD

**Issue by j-f1** [observablehq/parser#180]
Use case: Display an inline code block containing the literal text `<div ...${props} />` using a Markdown cell

Expected behavior: escape the `${}` with a ` \ `: `<div ...\${props} />` or `<div ...$\{props} />` or `<div ...\$\{props} />`

Actual results:
- `<div ...\${props} />` tries to evaluate `props`
- `<div ...$\{props} />` outputs `<div ...$\{props} />`
- `<div ...\$\{props} />` outputs `<div ...\$\{props} />`

**Comment by mbostock** [comment]
Our Markdown implementation doesn’t currently support Hypertext Literal; it just does naive string interpolation unless the interpolated value is a DOM element or an array. We plan on upgrading our Markdown implementation but we can only do that when we have standard library versioning.

So in the meantime, you should use Hypertext Literal if you want this. You can do that either in an HTML cell like so:

```html
Hello <span ${props}>world
```

Or by interpolating Hypertext Literal into Markdown like so:

```md
Hello ${htl.html`<span ${props}>world`}
```

Where

```js
props = ({style: {color: "red"}})
```

**Comment by j-f1** [comment]
I think you might be misunderstanding— I **don’t** want to interpolate `props`, I want the literal text `${props}` to appear in the output.

**Comment by mbostock** [comment]
Ah, sorry for misunderstanding. That does seem like a bug. Let me investigate further.

**Comment by mbostock** [comment]
Fixed in #181… but we have some additional downstream fixes before this will be fixed in production. Thanks for the report!
